int8 mode
bool stable_mode
float64 yaw_speed
float64 pitch_speed
float64 yaw_position
float64 pitch_position
